lib/Swagger/src/ContentEntity/Models/ResponseObj.php

```diff
@@ -63,7 +65,7 @@ class ResponseObj implements ArrayAccess
       */
     protected static $swaggerTypes = array(
         'title' => 'string',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)